flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/gateway/local/LocalExecutorITCase.java

```diff
@@ -444,6 +444,7 @@ public void testStreamQueryExecutionSink() throws Exception {
         replaceVars.put("$VAR_SOURCE_SINK_PATH", csvOutputPath);
         replaceVars.put("$VAR_UPDATE_MODE", "update-mode: append");
         replaceVars.put("$VAR_MAX_ROWS", "100");
+        replaceVars.put("$VAR_RESULT_MODE", "table");
```

Comment:
This test will create a `ResultDescriptor` that needs to read `sql-client.execution.result-mode` from the configuration. Because the value type of the option is enum and `$VAR_RESULT_MODE` doesn't belong to the enum.

In the origin code, it just reads from the `Envrionemnt` and compare the type whether is equal to `tableau`.